docs/tutorials/developer-journey/level-1/1.4-using-cycles.md

```diff
@@ -0,0 +1,229 @@
+# 1.4: Acquiring and using cycles
+
+## Overview
+
+If you recall from the previous module, [Internet Computer terminology](../level-0/02-ic-terms.md), cycles are used to measure the resources, such as memory, storage, and compute power, that are used by a canister. When a canister is deployed on the mainnet, cycles are 'charged' for every action that a canister performs.
+
+To obtain cycles, the IC's utility token, ICP, can be converted into cycles and transferred into a canister to be used to pay for that canister's consumed resources. ICP can be converted to cycles using a conversion rate where the current price of ICP measured is in [XDR](https://internetcomputer.org/docs/current/references/glossary#xdr), and one trillion cycles corresponds to one XDR. 
+
+Since cycles are not a currency and are only used to pay for a canister's consumed resources, developers manage the distribution of cycles through a special canister called a **cycles wallet**. When a canister needs to use the cycles stored in the cycles wallet, the canister's operations are executed using the canister principal of the cycles wallet, rather than your user principal. 
```

Comment:
I'd put that differently. The reason for having a cycles wallet canister, is that users principals can't hold cycles directly, only canisters can have cycles balances. So when you call a method that needs cycles from dfx you need to proxy this call via the cycles wallet to attach the necessary cycles. 

Each canister has its own cycles balance and doesn't use the cycles  balance of the developer's cycles wallet, but if you e.g. want to create a new canister, then this call needs to go through the cycles wallet to attach enough cycles to create the new canister and top up it's balance.

Reply:
Btw. did we explain "principal" already? It might make sense to link back to an explanation because this term is very IC specific.